dynamic_programming/longest_palindromic_subsequence.py

```diff
@@ -0,0 +1,56 @@
+"""
+Longest Palindromic Subsequence Problem:
+Find the longest sequence of characters from a given input string that
+is the same backwards and forwards
+https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
+"""
+
+
+def LongestPalindromicSubsequence(string, n):
+
+    """
+    Computes the longest Palindromic sequence of a string
+
+    :param string: str, the string we provide
+    :param n: int, the length of the string
+    :return L[0][n - 1]: int, the length of the longest Palindromic subsequence
+
+
+
+    >>> LongestPalindromicSubsequence("ABBCDABBC",9)
+    5
+    >>> LongestPalindromicSubsequence("ABACCG",6)
+    3
+    >>> LongestPalindromicSubsequence("55055901565109",14)
+    9
+    """
+
+    # creating an array to store the values generated
+    L = [[1 for i in range(n)] for i in range(n)]
```

Comment:
Uppercase is reserved for constant literals in Python.

Reply:
Sorry for the mistakes, I will fix the issues.